raiden/storage/serialization/serializer.py

```diff
@@ -17,6 +17,31 @@
 from raiden.storage.serialization.types import SchemaCache
 from raiden.utils.typing import Any, Dict
 
+MESSAGE_NAME_TO_QUALIFIED_NAME = {
+    "AuthenticatedMessage": "raiden.messages.abstract.AuthenticatedMessage",
```

Comment:
:rocket:  hyper-super-ultra nitpick :rocket: : Could you sort those alphabetically? Especially if your editor can do it easily.

Reply:
Sure, piping through `sort` is easy enough.